src/tests/ftest/util/recovery_test_base.py

```diff
@@ -0,0 +1,50 @@
+"""
+  (C) Copyright 2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import os
+from ClusterShell.NodeSet import NodeSet
+
+from apricot import TestWithServers
+from general_utils import run_pcmd
+
+
+class RecoveryTestBase(TestWithServers):
+    # pylint: disable=no-member
+    """Recovery test cases.
+
+    Test Class Description:
+        Used for recovery tests.
+
+    :avocado: recursive
+    """
+
+    def get_vos_file_path(self, pool):
+        """Get the VOS file path.
+
+        If there are multiple VOS files, returns the first file obtained by "ls".
+
+        Args:
+            pool (TestPool): Pool.
+
+        Returns:
+            str: VOS file path such as /mnt/daos0/<pool_uuid>/vos-0
+
+        """
+        hosts = NodeSet(self.hostlist_servers[0])
+        scm_mount = self.server_managers[0].get_config_value("scm_mount")
+        vos_path = os.path.join(scm_mount, pool.uuid.lower())
+        command = " ".join(["sudo", "ls", vos_path])
+        cmd_out = run_pcmd(hosts=hosts, command=command)
+
+        # return vos_file
+        for file in cmd_out[0]["stdout"]:
```

Comment:
It's better to use run_remote than run_pcmd. Something like this...
```suggestion
        from run_utils import run_remote
        cmd_out = run_remote(self.log, hosts, command)

        # return vos_file
        for file in cmd_out.output[0].stdout:
```

Reply:
Done. Thanks.